packages/schema/src/lib.rs

```diff
@@ -36,6 +36,7 @@ pub use remove::remove_schemas;
 /// }.render();
 /// ```
 pub use cosmwasm_schema_derive::generate_api;
+pub use cosmwasm_schema_derive::msg;
```

Comment:
Can you add a nice doc for this as well?

Reply:
Oh yeah, surprised I forgot to do that. On it!